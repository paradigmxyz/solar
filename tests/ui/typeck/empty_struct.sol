struct A {} //~ ERROR: structs must have at least one field