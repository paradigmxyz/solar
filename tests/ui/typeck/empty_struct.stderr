error: structs must have at least one field
  --> ROOT/tests/ui/typeck/empty_struct.sol:LL:CC
   |
LL | struct A {}
   |        ^
   |

error: aborting due to 1 previous error

